src/app/xtdb_contrib.clj

```diff
@@ -11,6 +11,6 @@
                       #(.close listener))))
     (m/reductions {} (xt/latest-completed-tx !xtdb)) ; initial value is the latest known tx, possibly nil
     (m/relieve {})
-    (m/latest (fn [{:keys [:xt/tx-time] :as ?tx}]
+    (m/latest (fn [{:keys [:xt.api/tx-time] :as ?tx}]
```

Comment:
Hey @noonian ,
shouldn't it be `xtdb.api/tx-time`? so it could be `::xt/tx-time` here?

It's easy to verify in REPL:
```clojure
(ns user
  (:require [xtdb.api :as xt]))

(comment
  (let [node (xt/start-node {})]

    (->> (xt/submit-tx node [[::xt/put {:xt/id (random-uuid)}]])
         (xt/await-tx node))
    (xt/latest-completed-tx node)
  ;; => #:xtdb.api{:tx-time #inst "2023-06-02T23:48:40.923-00:00", :tx-id 0}

    (::xt/tx-time (xt/latest-completed-tx node))
  ;; => #inst "2023-06-02T23:49:38.768-00:00"
    ))
```

Reply:
Good catch! I intended to use the fully qualified version to be more explicit what the key was but I still typoed the keyword. I'll fix the PR.